.github/workflows/release.yml

```diff
@@ -26,9 +26,36 @@ jobs:
         with:
           fetch-depth: 0
       - uses: olafurpg/setup-scala@v13
-      - run: sbt ci-release
+
+      - name: Run sbt ci-release (produces war as well)
+        run: sbt ci-release
         env:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+
+      - name: Find WAR file
+        id: find_war
+        run: echo "WAR_PATH=$(find . -name '*.war' | head -n 1)" >> $GITHUB_ENV
+
+      - name: Create Draft Release
+        id: create_draft_release
+        uses: actions/create-release@v1
```

Comment:
this action is actually no longer a live project, it's archived since March 2021.

There are, however, good alternatives depending on what we want:
* https://github.com/ncipollo/release-action
* https://github.com/softprops/action-gh-release (most popular)
* https://github.com/marvinpinto/action-automatic-releases
(I didn't check details about those)

Reply:
I like the initiative though!

Reply:
Latest version for origin is v1.1.4

Reply:
I picked - [softprops/action-gh-release](https://github.com/softprops/action-gh-release) as it seems it does everything we need. 

- body for release notes
- upload files
- fail on missing files
- tag name
- release name

Reply:
good good, we'll see if it works end to end, but agree it looks like a very sensible alternative